cypress/integration/console/applications/edit.spec.js

```diff
@@ -65,6 +65,20 @@ describe('Application general settings', () => {
       .should('be.visible')
   })
 
+  it('suceeds at changing skip payload crypto', () => {
+    cy.visit(`${Cypress.config('consoleRootPath')}/applications/${applicationId}/general-settings`)
+
+    cy.findByLabelText('Skip payload encryption and decryption').click()
+
+    cy.findByRole('button', { name: 'Save changes' }).click()
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText(`Application updated`)
+      .should('be.visible')
```

Comment:
```suggestion
      .findByText(`Application updated`)
      .should('be.visible')
```
I know we use the assertions like that in other places, but this is actually an unnecessary duplicate check, so let's not do that.